packages/web/src/components/gcds-pagination/gcds-pagination.css

```diff
@@ -0,0 +1,199 @@
+:host {
+  display: block;
+
+  ul {
+    padding: 0;
+    width: fit-content;
+    list-style: none;
+
+    li {
+      margin: var(--gcds-spacing-100);
```

Comment:
Remove this margin for the previous and next buttons when they are not displayed. It will give the component a little bit more space on smaller screen sizes.